OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -4,84 +4,97 @@
+# Pull serverjre:8 base image
 # -----------------------------------
 FROM oracle/serverjre:8
 
 # Maintainer
 # ------------------------------------
-MAINTAINER Hemastuti Baruah <hemastuti.baruah@oracle.com>
+MAINTAINER Prabhat Kishore <prabhat.kishore@oracle.com>
 
 # Environment variables required for this build (do NOT change)
 # ----------------------------------------------
-ENV OHS_PKG=fmw_12.2.1.0.0_ohs_linux64_Disk1_1of1.zip \
-    OHS_BIN=fmw_12.2.1.0.0_ohs_linux64.bin \
+ENV OHS_PKG=fmw_12.2.1.2.0_ohs_linux64_Disk1_1of1.zip \
+    OHS_BIN=fmw_12.2.1.2.0_ohs_linux64.bin \
     ORACLE_HOME=/u01/oracle/ohssa \
     USER_MEM_ARGS="-Djava.security.egd=file:/dev/./urandom" \ 
-    PATH=$PATH:/usr/java/default/bin:/u01/oracle/ohssa/oracle_common/common/bin
+    PATH=$PATH:/usr/java/default/bin:/u01/oracle/ohssa/oracle_common/common/bin \
+    NM_PORT=5556 \
+    OHS_LISTEN_PORT=7777 \
+    OHS_SSL_PORT=4443 \
+    JAVA_HOME=/usr/java/default \
+    MW_HOME=/u01/oracle/ohssa \
+    DOMAIN_NAME=ohsDomain \
+    OHS_COMPONENT_NAME=ohs_sa1 \
+    PATH=$PATH:$ORACLE_HOME/oracle_common/common/bin:$ORACLE_HOME/user_projects/domains/ohsDomain/bin:/u01/oracle/container-scripts
+
 
 # Copy packages to the new drive
 # -------------------------------------------------------
 COPY $OHS_PKG install.file oraInst.loc /u01/
 
-# Install OL7 required packages. Refer FMW 12.2.1 System requirements guide for complete list of packages
+# Install OL7 required packages. Refer FMW 12.2.1.2 System requirements guide for complete list of packages
 # Change the open file limits and kernel parameters that need changing
 # Setup filesystem and oracle user
 #---------------------------------------------------------------------------------------------------------
 # Proxy needs to be set before running yum in /etc/yum.conf
 RUN yum install -y \
-        unzip tar nfs-utils binutils compat-libcap1 compat-libstdc++ \
-        gcc gcc-c++ glibc glibc-devel libaio libaio-devel libgcc \
-	libstdc++ libstdc++-devel ksh make ocfs2-tools sysstat && \
+        unzip libaio sysstat && \
     yum clean all && \
-    rm -rf /var/cache/yum/* && \
     chmod a+xr /u01 && \
     useradd -b /u01 -m -s /bin/bash oracle && \
-    echo oracle:oracle | chpasswd && \
-    mkdir /u01/oracle/.inventory
+    mkdir /u01/oracle/.inventory && \
+    chown oracle:oracle -R /u01 
 
+# Install and configure Oracle JDK
 # Go to /u01 as user 'oracle' to proceed with OHS installation
 #--------------------------------------------------------
-RUN unzip -q -d /u01 /u01/$OHS_PKG && \
-    chown oracle:oracle -R /u01 && \
-    su -c "/u01/$OHS_BIN -ignoreSysPrereqs -novalidation -silent -responseFile /u01/install.file -invPtrLoc /u01/oraInst.loc -jreLoc $JAVA_HOME ORACLE_HOME=$ORACLE_HOME" - oracle && \
-    rm /u01/$OHS_PKG /u01/$OHS_BIN /u01/fmw_*_readme.htm /u01/oraInst.loc /u01/install.file
-
 USER oracle
-WORKDIR /u01/oracle/
+RUN cd /u01 && unzip -q /u01/$OHS_PKG && cd - && \
```

Comment:
You have Java installed, so you can drop the requirement to install `unzip` and run `java xf` instead. This will reduce the size of the image too.